test/image/mocks/image_non_numeric.json

```diff
@@ -0,0 +1,16 @@
+{
+  "data": [{
+    "type": "image",
+    "hoverinfo": "all",
+    "z": [
+      [
+        [true, true, true],
+        [false, false, false]
```

Comment:
True, there's something nice about ensuring we see the right axis ranges even when *all* of the pixels are invalid. But perhaps then a second mock (or a second subplot on the same one?) that includes a couple of valid pixels as well as various kinds of invalid pixels - any single channel missing for example.

Reply:
Good call. That mock should be improved.

Reply:
Done in https://github.com/plotly/plotly.js/pull/4325/commits/d6abde903a8988edc218b63498381d9d2fa472a5.